### PR TITLE
document fetchOptions

### LINKED ISSUE #200: [NEW]: fetchOptions in TS SDK
### Priority

Low

### Description

Document the [fetchOptions](https://github.com/xataio/client-ts/blob/main/packages/client/src/schema/query.ts#L86) parameter

### How will this impact users?

From Next.js docs on [Caching Data](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#caching-data):
fetch requests that use the POST method are also automatically cached. Unless it's inside a Route Handler that uses the POST method, then it will not be cached.
Since queries use POST, it means results from Xata that are not performed from route.ts  are actually cached.

### Context

https://discord.com/channels/996791218879086662/1188892129259438091/1188892129259438091

### Suggested resolution

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kostasb**
Closes #200

### CODE REVIEW

File: 040-SDK/010-TypeScript/020-configuration.mdx [hunk 1]
```diff
@@ -38,6 +38,24 @@ const xata = new XataClient({ apiKey: XATA_API_KEY });
 
 In case your runtime does not provide a built-in Fetch API (such as with versions prior to Node 18), you will need to pass a fetch implementation to the `fetch ` constructor parameter of the XataClient in the `xata.ts` / `xata.js `file in your project.
 
+## Passing options to fetch
+
+Some frameworks, like Next.js, support additional fetch options which allow you to configure the caching and revalidating behavior for each fetch request on the server.
```
Comment (joan-ing) [inline 040-SDK/010-TypeScript/020-configuration.mdx:43]:
```suggestion
Some frameworks, such as Next.js, support additional fetch options. These settings allow you to specifically control how caching and revalidation are handled for each fetch operation on the server.
```

**Review by joan-ing — APPROVED** [review]
lgtm

**Comment by kostasb** [comment]
Let's not merge this yet. While fetchOptions does its part from the SDK side, we haven't been able to verify the cache actually works with these.

File: 040-SDK/010-TypeScript/020-configuration.mdx [hunk 2]
```diff
@@ -38,6 +38,24 @@ const xata = new XataClient({ apiKey: XATA_API_KEY });
 
 In case your runtime does not provide a built-in Fetch API (such as with versions prior to Node 18), you will need to pass a fetch implementation to the `fetch ` constructor parameter of the XataClient in the `xata.ts` / `xata.js `file in your project.
 
+## Passing options to fetch
+
+Some frameworks, such as Next.js, support additional fetch options. These settings allow you to specifically control how caching and revalidation are handled for each fetch operation on the server.
+
+The Xata SDK supports the `fetchOptions` parameter for passing options to fetch.
+
+This parameter can be used to configure cache revalidation at a timed interval:
```
Comment (SferaDev) [inline 040-SDK/010-TypeScript/020-configuration.mdx:47]: I'd make this more clear it's only for Next.js

Comment (kostasb) [reply 040-SDK/010-TypeScript/020-configuration.mdx:47]: Will add clarifications once we have findings from https://github.com/xataio/client-ts/issues/1311 as it doesn't seem to actually work as documented, maybe there's more steps needed in Next.js.

**Comment by kostasb** [comment]
I will close this PR until we figure out additional guidelines required from https://github.com/xataio/client-ts/issues/1311 , as merging as-is is unlikely to help users.

**Pull request closed without merging**